xmtp_api_d14n/src/endpoints/d14n/query_envelopes.rs

```diff
@@ -25,7 +25,7 @@ impl Endpoint for QueryEnvelope {
     type Output = QueryEnvelopesResponse;
 
     fn http_endpoint(&self) -> Cow<'static, str> {
-        todo!()
+        Cow::from("/xmtp.xmtpv4.message_api.ReplicationApi/QueryEnvelopes")
```

Comment:
I think these endpoints are wrong, should be `mls/v2/query_envelopes` according to the [proto](https://github.com/xmtp/proto/blob/d1da51b50118819fbd12c31ac57a1759f27ad990/proto/xmtpv4/message_api/message_api.proto#L81) ('post' field)